src/pages/en/community/rules.mdx

```diff
@@ -86,9 +86,12 @@ Do not change your avatar, nickname, username or custom/rich status to try to ev
 
 #### Nicknames
 
-All nicknames must adhere to the rules above. Additionally, they should not be hard to read or too noisy. Nicknames may be considered difficult to read or noisy if they contain emojis, Zalgo or other difficult-to-read styling, or Unicode control characters. "Empty" nicknames are also considered difficult to read.
+All nicknames must adhere to the rules above. Additionally, nicknames have to adhere to the following rules:
+1. Nicknames must be easy to read, especially by a screen-reader (ex: no Zalgo or otherwise hard to read non-ascii characters, with the exception of non-Latin names)
+2. Nicknames cannot be intentionally confusing or misleading (ex: impersonation is stricly forbidden, as well as using similar looking nicknames to be intentionally confusing)
+3. Nicknames must have some readable characters (ex: no "empty" nicknames)
 
-We also reserve the right to change or reset nicknames that are designed to place you higher in the member list in community spaces that have one, which is known as "hoisting".
+We also reserve the right to change or reset nicknames that are infringing those rules, or designed to place you higher in the member list in community spaces that have one, which is known as "hoisting".
```

Comment:
Maybe we should just put no hoisting in the rules to make the rule set more consistent